src/site/topic12.rst

```diff
@@ -11,6 +11,57 @@ Idea #2
 Idea #3
 =======
 
+
+Expand Capacity
+---------------
+
+.. image:: img/arrayqueue_expand0.png
+   :width: 500 px
+   :align: center
+
+* Suppose we ``enqueue`` one more element
+
+.. image:: img/arrayqueue_expand1.png
+   :width: 500 px
+   :align: center
+
+* Now the queue is full
+    * ``size == queue.length``
+
+* If we ``enqueue`` again, we'll need to call ``expandCapacity``
+* **However** we can't just double the size of the array and copy the contents over like we did before
```

Comment:
I'd expand of why here. What stateful values are coupled to the array? May be beneficial to go back to the stack example and see why `top` is still correct after the expansion.

Reply:
Grand idea.

Reply:
Although, depending on how much detail I go into with it and also the stack example, this may be best as an aside.

Reply:
On additional thought, I think an aside may be best as it will give more of a feel for fleshing out the details. #139